pkgs/applications/networking/cluster/zarf/default.nix

```diff
@@ -1,20 +1,20 @@
 { lib
-, buildGoModule
+, buildGo121Module
 , fetchFromGitHub
 }:
 
-buildGoModule rec {
+buildGo121Module rec {
```

Comment:
why did this change? I do not see the need

Reply:
Great question... I'll change it back. It was not working until I specified Go121 specifically, but now it appears to be working... hmm